### PR TITLE
Added exclude argument to persistently exclude sites.

### PR DESCRIPTION
**Pull request by peterjschroeder**

### CODE REVIEW

File: googler [hunk 1]
```diff
@@ -3510,6 +3515,8 @@ def parse_args(args=None, namespace=None):
            help='ending date/month/year of date range; see --from')
     addarg('-w', '--site', dest='sites', action='append', metavar='SITE',
            help='search a site using Google')
+    addarg('-e', '--exclude', dest='exclude', action='append', metavar='EXCLUDE',
```
Comment (jarun) [inline googler:3518]: Please add this option in the readme and the auto-completion scripts as well.

**Review by jarun — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -286,6 +286,8 @@ optional arguments:
                         and overrides -t, --time
   --to TO               ending date/month/year of date range; see --from
   -w SITE, --site SITE  search a site using Google
+  -e EXCLUDE, --exclude EXCLUDE
```
Comment (jarun) [inline README.md:289]: This should also be SITE (instead of EXCLUDE). I will take care of it.

**Comment by jarun** [comment]
Thank you!

**Comment by peterjschroeder** [comment]
You're welcome.

**Comment by jarun** [comment]
All good at commit f62d3cd718ff8fa65d9f5d9d38b563784114eab8.

**Pull request merged**